app/bundles/CoreBundle/Assets/js/libraries/4j.codemirror.js

```diff
@@ -3586,7 +3586,11 @@
     }
     if (clickInGutter(cm, e)) return;
     var start = posFromMouse(cm, e);
-    window.focus();
+
+    if (typeof Mautic != 'undefined' && Mautic.codeMode != true) {
+      console.log(Mautic);
```

Comment:
Need to update this as the JS fixes for dwc are not longer required - fixed in another PR by @virlatinus